.vscode/settings.json

```diff
@@ -51,5 +51,8 @@
         "description": "Test pattern or filename"
       }
     ]
+  },
+  "[typescriptreact]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
Shouldn't be a part of this change, please remove.

Reply:
Weird, I thought that the settings.json would be on the Git Ignore list, will remove rn.

Reply:
Another thing that is non-intuitive with this that I just realized, is that when you modify the `settings.json`, it does not appear in the Source Control Tab of VSCode, which is interesting. Weird that they have an inconsistency between files changed there and in the actual PR.